docs/connect.html

```diff
@@ -58,10 +58,13 @@
 <ul>
     <li><code>group.id</code> (default <code>connect-cluster</code>) - unique name for the cluster, used in forming the Connect cluster group; note that this <b>must not conflict</b> with consumer group IDs</li>
     <li><code>config.storage.topic</code> (default <code>connect-configs</code>) - topic to use for storing connector and task configurations; note that this should be a single partition, highly replicated topic</li>
-    <li><code>offset.storage.topic</code> (default <code>connect-offsets</code>) - topic to use for ; this topic should have many partitions and be replicated</li>
+    <li><code>offset.storage.topic</code> (default <code>connect-offsets</code>) - topic to use for storing offsets; this topic should have many partitions and be replicated</li>
+    <li><code>status.storage.topic</code> (default <code>connect-status</code>) - topic to use for storing statuses; this topic can have multiple partitions and be replicated</li>
 </ul>
 
-Note that in distributed mode the connector configurations are not passed on the command line. Instead, use the REST API described below to create, modify, and destroy connectors.
+We recommend to manually create the topics for offset, configs and statuses in order to achieve the desired the number of partitions and replication factor. If the topics are not yet created when starting Kafka Connect, the topics will be auto created with default number of partitions and replication factor, which may not be best suited for its usage. 
```

Comment:
I blanked on this when @Ishiihara asked me about it earlier, but reviewing the code I can remember the reason why now: we need ordering across messages with different keys, specifically for committing tasks. If the task configs and the commit message end up on different partitions then the ordering isn't guaranteed and we may not get correct behavior.

In the original implementation, I went back and forth on whether to just bunch the updates together as a single message or separate them by key. Not sure we ended up making the right decision (and it might be something we want to revisit), but that is the reason for now that we need to have 1 partition.

(Additionally, I just find a single partition more intuitive because of how I think about the config topic -- as a single, ordered sequence of updates. It's also a low traffic topic, so we don't really expect to get any benefit from more partitions. That said, I suppose it does incur some cost in terms of having to manually create the topic...)